backend/README.md

```diff
@@ -35,7 +34,8 @@ The `SECRET_KEY` should be sufficiently random. For example, you can use the fol
 
 Update the `DB_*` variables as needed, so they use the prepared PostgreSQL installation.
 
-4. Run `python manage.py makemigrations`
-5. Run `python manage.py migrate`
+3. Run `docker compose run django python manage.py makemigrations`
+4. Run `docker compose run django python manage.py migrate`
+4. Run `docker compose run django python manage.py collectstatic`
 
-From now on, you can use the command `python manage.py runserver` to have the Django backend run.
\ No newline at end of file
+From now on, you can use the command `docker compose up -d --build` to have the Django backend run.
```

Comment:
```suggestion
From now on, you can use the command `sudo docker compose up -d --build` to have the Django backend run.
```

Reply:
why do you need sudo? :)

Reply:
I get a Permission denied when I run the command, because docker wants to create the directory ~/.docker/buildx/instances

Reply:
ah, interesting. Okay then